browser.js

```diff
@@ -1,4 +1,5 @@
 var createHmac = require('create-hmac')
+var MAX_INT = Math.pow(2, 30) - 1 // default in iojs
```

Comment:
Poor name it's not maximum integer but maximum buffer allocation, but since
this is a sync error in node we have to pick a number

On Thu, Mar 26, 2015, 10:29 PM Daniel Cousens <notifications@github.com>
wrote:

> In browser.js
> <https://github.com/crypto-browserify/pbkdf2/pull/12#discussion_r27272718>
> :
>
> > @@ -1,4 +1,5 @@
> >  var createHmac = require('create-hmac')
> > +var MAX_INT = Math.pow(2, 30) - 1 // default in iojs
>
> How are we proving that keylen will never be greater than this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crypto-browserify/pbkdf2/pull/12/files#r27272718>.
>